src/App.tsx

```diff
@@ -148,6 +152,11 @@ function App() {
           path: "change-password",
           element: <ChangePassword />,
         },
+        {
+          path:'booking-page/:id',
+        element:<BookingPage/>
```

Comment:
Generally, I don't reccomend adding Page to the page component as it is a little bit redundant since we are using it in the routes.